spyder/config/gui.py

```diff
@@ -38,9 +36,6 @@
 # Stylesheet to remove the indicator that appears on tool buttons with a menu.
 STYLE_BUTTON_CSS = "QToolButton::menu-indicator{image: none;}"
 
-# Check for old PyQt versions
-OLD_PYQT = programs.check_version(PYQT_VERSION, "5.12", "<")
```

Comment:
This requires an update to `pyqt >=5.15,<5.16`, which I plan to do in another PR.

Reply:
For tracebility, I guess the PR is https://github.com/spyder-ide/spyder/pull/21573

Reply:
Correct.